docs/reference/api/connectivity/lorawan/loraradio.md

```diff
@@ -0,0 +1,12 @@
+<h2 id="loraradio-api">LoRaRadio</h2>
+
+`LoRaRadio` is a pure virtual class that defines APIs for a LoRa radio driver.
+
+An implementation of this class is passed to the Arm Mbed LoRaWAN stack. Please see existing [Mbed OS supported drivers](https://github.com/ARMmbed/mbed-semtech-lora-rf-drivers) for more information.
+
+### LoRaRadio class reference
+
+[![View code](https://www.mbed.com/embed/?type=library)](http://os-doc-builder.test.mbed.com/docs/v5.8/mbed-os-api-doxy/class_l_o_r_a_radio.html)
+
+### LoRaRadio example
```

Comment:
@AnttiKauppila Could you please communicate to Amanda through email the reason why we are not posting an embedded example ? I didn't not get hold of the full context when we were discussing this.

Reply:
@AnotherButler Okay, I got clearance to tell you the reason. Please note that we do have a very simple example code which do exist at os.mbed.com, https://os.mbed.com/teams/mbed-os-examples/code/lorawan-example/. But this is private. We can't make it public as it will need at least two months of process through legal and we don't have enough time for it. So for the time being it will suffice to just provide a link for the github repository which contains full example code.

Reply:
I'm fine with this going in without an example for now, but we should add an example as soon as possible.

Reply:
Hi Amanda, example is now public and ready for Online compiler

Reply:
This is valid link currently:
https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/class_lo_ra_radio.html